src/components/atoms/avatar/avatar.jsx

```diff
@@ -0,0 +1,3 @@
+import UIAvatar from '@material-ui/core/Avatar';
+
+export default UIAvatar;
```

Comment:
Currently this is just Material UI's avatar exported. Long-term, we don't want material UI because of how large it is so would be better to refactor into a custom styled component. This is isolated though so it's easy enough to do as a follow-up PR. Also needs tests + storybook.